### ISSUE TITLE
    _ious[cell_ind, a, :] = ious_reshaped[cell_ind, a, i]

### ISSUE THREAD

**Issue by 22wei22** [longcw/yolo2-pytorch#64]
Ask for help,in your code,you use _ious[cell_ind, a, :] = ious_reshaped[cell_ind, a, i],
 I think _ious[cell_ind, a, :] = anchor_ious[a, i]

**Comment by Erotemic** [comment]
No, the predicted IOU should be compared to the IOU with the groundtruth box, not the anchor box. I'm vetting this code fairly extensively right now, and I believe this is right.
